homeassistant/components/generic/config_flow.py

```diff
@@ -183,14 +183,31 @@ async def async_test_still(
     except (
         TimeoutError,
         RequestError,
-        HTTPStatusError,
         TimeoutException,
     ) as err:
         _LOGGER.error("Error getting camera image from %s: %s", url, type(err).__name__)
         return {CONF_STILL_IMAGE_URL: "unable_still_load"}, None
+    except HTTPStatusError as err:
+        _LOGGER.error(
+            "Error getting camera image from %s: %s %s",
+            url,
+            type(err).__name__,
+            err.response.text,
+        )
```

Comment:
If err.response.text does not contain the http response code - let's add it.

Reply:
`.text` already includes the error code. Example: `"500 Internal Server Error"`